docs/articles/documentation/test-api/actions/take-screenshot.md

```diff
@@ -21,12 +21,7 @@ t.takeScreenshot( [path] )
 
 Parameter           | Type   | Description
 ------------------- | ------ | -----------------------------------------------------------------------------------------------------
-`path`&#160;*(optional)* | String | The relative path and the name of the screenshot file to be created. Resolved from the *screenshot directory* specified by using the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option.
-
-By default, the path to which screenshots are saved is specified as:
-
-* `{currentDate}\test-{testIndex}\{userAgent}\{screenshotIndex}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is disabled;
-* `{currentDate}\test-{testIndex}\run-{quarantineAttempt}\{userAgent}\{screenshotIndex}.png` if the [quarantine mode](../../using-testcafe/command-line-interface.md#-q---quarantine-mode) is enabled.
+`path`&#160;*(optional)* | String | The screenshot file's relative path and name. The path is resolved relative to the root directory specified by the [runner.screenshots](../../using-testcafe/programming-interface/runner.md#screenshots) API method or the [screenshots](../../using-testcafe/command-line-interface.md#-s-path---screenshots-path) command line option. This path overrides the relative path specified by the default or custom [path patterns](../../using-testcafe/command-line-interface.md#path-patterns).
```

Comment:
- This topic describes how you can take screenshots of the tested page -> This topic describes how to take screenshots of the tested page.
- these actions require a ICCCM/EWMH-compliant window manager on Linux -> 'an', not 'a'
- Important! If the screenshot directory is not specified with the runner.screenshots API method or the screenshots command line option, the screenshot actions are ignored -> Important! Screenshot actions are ignored if the screenshot directory is not specified with the runner.screenshots API method or the screenshots command line option.
- The screenshot file's relative path and name. The path is resolved (resolved? relative to?) relative to the root directory the runner.screenshots API method or screenshots command line option specifies. This path overrides the relative path the default or custom path patterns specify.
- The following example shows how to use the t.takeScreenshot action**.** -> The following example shows how to use the t.takeScreenshot action**:**
- "Identifies the webpage element whose screenshot will be taken." This sentence does not make sense. 'element whose screenshot'? You also do not need to use the future tense.
- Options that define how the screenshot **will be** taken -> Options that define how the screenshot **is** taken.
- "By default, the path to which screenshots are saved is specified as:" -> Do not use 'By default,'
- The options object contains the following properties**.** -> The options object contains the following properties**:**

Reply:
>'element whose screenshot'

Yes, TestCafe provides functionality to take a screenshot of a separate element on a webpage. _Element_ is a term.